localization.lua

```diff
@@ -14,5 +14,13 @@ local locale = GetLocale()
 
 if locale == 'ruRU' then
     WowUpTexts.Okay = 'Окей'
-    WowUpTexts.updateNotification = 'Уведомление от WowUp: У вас есть %d модификаций нуждающихся в обновлении'
+    WowUpTexts.updateNotification = 'Уведомление от WowUp: У вас есть %d модификаций нуждающихся в обновлении',
```

Comment:
If you can remove the `,` at the end of the lines you've modified, it should be good to merge. The lines you've modified are variable assignments, and those don't have a character at the end to designate the end of the line